### PR TITLE
docs: update FAQ section

### LINKED ISSUE #256: Add New FAQs
## Description
Pieces' docs site is currently going through a major migration, so the FAQ sections needed to be updated

### PR DESCRIPTION
**Pull request by CBID2**
## Description
This PR updates the FAQ section.   

## Related Tickets and Documents

Closes #256 

## Notes to Reviewers

This is still a work in progress

### CODE REVIEW

**Comment by CBID2** [comment]
Hey @mason-at-pieces. Even though this is still a work-in-progress, there are some parts I need help with, so I added them as comments in my pull request. Check them out.

File: docs/faq.mdx [hunk 1]
```diff
@@ -3,8 +3,44 @@ title: Frequently Asked Questions
...
 
+## PiecesOS
+<Tabs defaultValue={"macos"}  values={[{label: 'macOS', value: 'macos'}, {label: 'Windows', value: 'windows'}, {label: 'Linux', value: 'linux'}]}>
+  <TabItem value="macos">
+    <CTAButton
+      href={"/faq/macos"}
+      label={'MacOS'}
+      icon={'/assets/mac_os.png'}
+      iconDark={'/assets/mac_os_dark.png'}
+      type={'secondary'}
```
Comment (CBID2) [inline docs/faq.mdx:21]: @mason-at-pieces, I'm trying to place the logo near the tab's title. Do you have any tips on how to do this?

File: docs/faq.mdx [hunk 2]
```diff
@@ -3,8 +3,44 @@ title: Frequently Asked Questions
...
+    <p>Linux</p>
+    <CTAButton
+      href={"/faq/linux"}
+      label={'Linux'}
+      icon={'/assets/linux_logo.png'}
+      type={'secondary'}
+    />
+  </TabItem>
+</Tabs>
 ## How can I discover useful snippets within the Pieces for Developers Desktop App?
```
Comment (CBID2) [inline docs/faq.mdx:44]: Not sure what to do with this and the other parts of this file @mason-at-pieces. If we add the content that @rosie-at-pieces mentioned in the [google doc](https://docs.google.com/document/d/1NFi5khaElnkpOBNwoApiGNtxYjylQ0lZrYNZKTVEJBA/edit), I'm concerned that users will suffer from cognitive overload.

Comment (mason-at-pieces) [reply docs/faq.mdx:44]: I think it should be ok. 

If you can just get all the content added for the first faq from the google doc, then we can refine how we want to format it and then apply that to all the rest of the new faqs.

File: docs/faq.mdx (hunk 1 above)
Comment (mason-at-pieces) [reply docs/faq.mdx:21]: Don't worry too much about this for now. Just get the content for the first faq from the google doc added and then we can refine things.

**Comment by mason-at-pieces** [comment]
@CBID2 Let me know if you need any help on this update. We want to keep things moving so we can get this in by the end of the month 👍

**Comment by CBID2** [comment]
> @CBID2 Let me know if you need any help on this update. We want to keep things moving so we can get this in by the end of the month 👍 

Sure @mason-at-pieces. I'm going to make some updates today

**Comment by CBID2** [comment]
Hey @mason-at-pieces. I got the content in for the Pieces OS section in the FAQs, I was unable to download the screenshots @rosie-at-pieces posted in the google doc, so can you add them in there please?

**Comment by mason-at-pieces** [comment]
> Hey @mason-at-pieces. I got the content in for the Pieces OS section in the FAQs

So there isn't a `Pieces OS` section, instead that is part of the `How do I check my Pieces versions and update the app?` section. Do you see how we have it structured on the doc?

**Comment by rosie-at-pieces** [comment]
@CBID2 just to clarify, are you using this document:

https://docs.google.com/document/d/1NFi5khaElnkpOBNwoApiGNtxYjylQ0lZrYNZKTVEJBA/edit?usp=sharing

**Comment by CBID2** [comment]
> @CBID2 just to clarify, are you using this document:
> 
> 
> 
> https://docs.google.com/document/d/1NFi5khaElnkpOBNwoApiGNtxYjylQ0lZrYNZKTVEJBA/edit?usp=sharing

Yes @rosie-at-pieces.

**Comment by CBID2** [comment]
Hi @mason-at-pieces. I made the adjustments you and @rosie-at-pieces suggested.

**Comment by mason-at-pieces** [comment]
Hey @CBID2, so we actually had to make some adjustments on our end and this resulted in us just completing this internally.

We are still figuring out the best way to have contributors on our docs so I appreciate your patience in this process!

My idea is that we have issues that we will complete internally but also ensure that we have issues open for contributors to work on. Would love your feedback on this idea as well 👍

**Comment by CBID2** [comment]
> Hey @CBID2, so we actually had to make some adjustments on our end and this resulted in us just completing this internally.
> 
> We are still figuring out the best way to have contributors on our docs so I appreciate your patience in this process!
> 
> My idea is that we have issues that we will complete internally but also ensure that we have issues open for contributors to work on. Would love your feedback on this idea as well if you'd like to open a discussion to chat about: https://github.com/pieces-app/opensource/discussions 👍
> 
> For now, I will close this pr and we can work on getting a better process in place to ensure we don't have more prs like this that we have to close for some reason that is our fault 👍

Hey @mason-at-pieces. I took your advice and created a discussion. Here it is: https://github.com/pieces-app/opensource/discussions/65#discussion-6427347

**Pull request closed without merging**